CallAutomation_SimpleIvr/readme.md

```diff
@@ -53,15 +53,23 @@ The application is an app service application built on .NET6.0.
 	- ACSAlternatePhoneNumber: Azure Communication Service acquired phone number.
 	- CallbackUriBase: URI of the deployed app service or Visual studio dev tunnel url.
 	- ParticipantToAdd: Target phone number to add as participant.
-
+	
 ### Create Webhook for Microsoft.Communication.IncomingCall event and Microsoft.Communication.RecordingFileStatusUpdated event
-1. Configure webhook from ACS events tab for incoming call event.
- 	- Manually configuring the webhook using this command. use above published "'https://<IncomingCallMediaStreaming-URL>/OnIncomingCall" URL as webhook URL.
+IncomingCall is an Azure Event Grid event for notifying incoming calls to your Communication Services resource. To learn more about it, see [this guide](https://learn.microsoft.com/en-us/azure/communication-services/concepts/call-automation/incoming-call-notification). 
+1. Navigate to your resource on Azure portal and select `Events` from the left side menu.
+1. Select `+ Event Subscription` to create a new subscription. 
+1. Filter for Incoming Call event. 
+1. Choose endpoint type as web hook and provide the public url generated for your application by ngrok. Make sure to provide the exact api route that you programmed to receive the event previously. In this case, it would be <ngrok_url>/api/incomingCall. 
+![Screenshot of portal page to create a new event subscription.](./media/event-susbcription.png)
+![Screenshot of portal page to create a new event subscription.](./media/MicrosoftTeams-image.png)
```

Comment:
Just curious why can not we have just one image ? any reason to split the image?